src/Foundation/Commerce/code/Models/CommerceUSerModel.cs

```diff
@@ -14,8 +14,11 @@
 
 namespace Wooli.Foundation.Commerce.Models
 {
+    using System;
+
     using TypeLite;
 
+    [Obsolete("Use Entity.Users.User")]
     [TsClass]
     public class CommerceUserModel
```

Comment:
Please remove this one, it shouldn't be used anymore

Reply:
It's used in AccountController and AccountRepository. They will be removed after refactoring.